worlds/ladx/LADXR/generator.py

```diff
@@ -374,12 +385,12 @@ def gen_hint():
                     TrendyGame.option_impossible: (3, 16),
                 }
                 def speed():
-                    return rnd.randint(*speeds[ap_settings["trendy_game"]])
+                    return world.random.randint(*speeds[world.options.trendy_game])
                 rom.banks[0x4][0x76A0-0x4000] = 0xFF - speed()
                 rom.banks[0x4][0x76A2-0x4000] = speed()
                 rom.banks[0x4][0x76A6-0x4000] = speed()
                 rom.banks[0x4][0x76A8-0x4000] = 0xFF - speed()
-                if int(ap_settings["trendy_game"]) >= TrendyGame.option_hardest:
+                if int(world.options.trendy_game) >= TrendyGame.option_hardest:

```

Comment:
A cast that isn't needed
```suggestion
                if world.options.trendy_game >= TrendyGame.option_hardest:
```